### DISCUSSION TITLE
Could this detector differentiate β− and β+?

### DISCUSSION THREAD

**Discussion by seanboyce** [ozel/DIY_particle_detector#52] [General]
I've put together a couple of your detectors and the design works wonderfully (I'll share some photos soon). I'm trying to think up some interesting experiments for a STEM program.

I've noticed that K-40 has an uncommon (0.001%) decay mode that emits a positron, in addition to the more common beta decay (89%) and electron capture (11%).

I think it would be neat if I could differentiate the β− and the β+, but it looks like the β+ is only around ~44keV. Would that even be likely to reach the diode at all (through the plastic covering) and generate a signal? Would the signal look any different, given the continuous distribution of β− energies in the more common decay mode?

I'm a bit new to particle physics, I couldn't find an obvious answer after some research. My best guess so far though is I'd need to design a SiPM based detector with a scintillator to measure the gamma ray energy from the positron annihilation (and wait a long time). Is that approximately correct, or is there an easier way I missed?

**Comment by SteveF161** [comment]
Hi Sean

I think your biggest problems would be the low arrival rate of beta+ and
distinguishing it from the beta- signal.

I used about 30g of KCl with my detector and got beta- rates of around
1,000 per hour. The beta+ rate is lower by a factor of 89,000 so that
equates to about 0.25 per day, and that would be indistinguishable from the
statistical noise level of about 150 in same interval for beta-

Nice idea though. Maybe schools can get access to some low-level positron
sources on educational grounds?

Regards

Steve

**Reply by seanboyce** [reply]
Aww, mains power fluctuation ate my first reply :(

Thanks for the quick reply!

I could reach out to some universities and see what the legality of possessing low-level sources is here in Vietnam. I will probably just have students calculate the positron emission rate of a banana -- that seems a bit more tangible. Back of the napkin is that it's some reasonably interesting number.

I have some other experiments planned -- I'll describe them in case you are interested in knowing how your device is being used 'in the wild':

Level monitoring for background vs. KCl vs banana. Also petrol -- I suspect that the distillation process concentrates radon, and the decay products might be detectable (but I don't know yet).

I'll buy some rice from Fukushima and compare to local rice. Also, since some nearby countries have banned Japanese seafood over water released at Fukushima, I can compare that too. I don't expect to find a significant difference, but the point is more to get students thinking critically about nuclear technology, and maybe consider a career in it, instead of just reacting with fear. 

I'm handy in AVR assembly, so I wrote a von-Neumann whitewashing algorithm that produces an unbiased outcome based on 3 particle detection events (this was for a previous alpha spectrometer I built in 2008 or something). The purpose is to create something like the Schrödinger's Cat experiment (at least approximately enough to discuss entropy and random number generation), but with less fiddly timing. Obviously, I wouldn't risk the cat. Instead, a coffee machine will use either caffeinated or decaffeinated coffee.

Or maybe I'll serve a piece of cake, and a 50% probability of another piece of cake under a lid. So you can simultaneously have your cake, and eat it too. I think I'll have to think that one out a little more for factual accuracy.

**Reply by ozel** [reply]
Hi Sean, the main problem has already been pointed out by Steve: the beta+ decay rate is extremely low and most of it is absorbed or annihilated within the potassium salt itself before it reaches the detector. 

In general, the DIY detector has high directionality as the sensitive volume of the diodes is rather flat (~ 50 um x 2.7 mm x 2.7mm) and small. And since it is a solid-state sensor the detection efficiency for _charged_ particles is basically 100% as long as the energy is above the 33 keV threshold (this is for the alpha-spectrometer circuit/part values. The electron detector got a somewhat higher energy threshold). While the tiny diodes aren't ideal for measuring low contamination of spread-out radioactive matter, they could be used in a smart way in setups where directionality is important. The beta+ detection is typically done with multi-coincidence measurements, the first paper describing the discovery in the 60s used a liquid scintillator plus a duo of voluminous crystal scintillators for the resulting gamma pair. The K40 decay modes are quite interesting, some aspects of the rare decays aren't fully solved. There seems to be a ten times higher energy beta+ decay mode that is even rarer than the better-known 44 keV one: https://gchron.copernicus.org/articles/2/355/2020/

Since you mentioned you built several of my detectors, I think it should be easy to record the angular deflection of betas (electrons!) in a strong (neodym) magnetic field. This experiment was always on my todo list, but I have had no time to try it. 
Even if the overall decay rate from 10 grams of KCL is low ( 1 g(𝐾𝐶𝑙) = 16.25 Bq), I would expect that you should rather quickly measure a detection rate difference/gradient when you arrange 3 DIY electron detectors around a magnet (at some distance to the magnet to account for flight path). Mounting the detector PCBs upright, you can almost fully exclude any detection of cosmic muons. 
Here is an example of a beta deflection measurement setup from Phywe, detailed instructions can be downloaded:
https://www.phywe.com/versuche-sets/schuelerversuche/ablenkung-von-beta-strahlen-im-magnetfeld-mit-cobra-smartsense_11147/

I do appreciate the sharing of your exercise ideas, together with a colleague we also played a bit with von Neumann and biased/unbiased coin-flipping with students as well as physics teachers. A very different but fun activity related to decay rates and Poisson distribution is listening to & recording rainfall:
https://indico.cern.ch/event/572852/contributions/2574359/attachments/1487162/2310029/Team_Raindrops_and_Muons.pdf
Only certain rain conditions exhibit this distribution feature, after some trials we succeeded by aiming water from a garden hose at a rather low flow rate (via a parabola-like path) on an umbrella and recording from below the umbrella. Further context: https://amt.copernicus.org/preprints/amt-2020-174/amt-2020-174.pdf

Cheers,
Oliver

**Reply by seanboyce** [reply]
Oh, I like the idea of measuring beta deflection, thanks for sharing that. Nd magnets are very inexpensive here too so it's quite practical.

I have some experience building sensitive piezo vibration sensors -- e.g. a piezo element and an opamp or CD4069 hex inverter. The latter is nice because you ought to get pretty clean 5V pulses out for water droplets hitting a metal surface.

After playing around with the detectors a bit, and reading the comments here, I have a much better idea of what experiments are practical to run :)
